pages/common/hugo.md

```diff
@@ -0,0 +1,31 @@
+# hugo
+
+> Hugo is a Fast and Flexible Static Site Generator.
+
+- Create a new hugo site:
+
+`hugo new site {{path/to/site}}`
+
+- Create a new hugo theme (you may also download one from [https://themes.gohugo.io/]()):
+
+`hugo new theme {{theme_name}}`
+
+- Create a new page:
+
+`hugo new {{section_name}}/{{filename}}{{.ext}}`
```

Comment:
I don't think it's necessary to have the extension as a separate token -- see https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#token-syntax for details.

Simply `{{section_name}}/{{filename}}` should work IMO, or at most `{{section_name}}/{{filename.ext}}`